soc/nxp/imxrt/imxrt118x/soc.c

```diff
@@ -283,6 +283,31 @@ static ALWAYS_INLINE void clock_init(void)
 
 #endif /* CONFIG_MCUX_ACMP */
 
+#ifdef CONFIG_CAN_MCUX_FLEXCAN
+
+#if DT_NODE_HAS_STATUS(DT_NODELABEL(flexcan1), okay)
+	/* Configure CAN1 using MuxSysPll3Out */
+	rootCfg.mux = kCLOCK_CAN1_ClockRoot_MuxSysPll3Out;
+	rootCfg.div = 6;
+	CLOCK_SetRootClock(kCLOCK_Root_Can1, &rootCfg);
+#endif
+
+#if DT_NODE_HAS_STATUS(DT_NODELABEL(flexcan2), okay)
+	/* Configure CAN2 using MuxSysPll3Out */
+	rootCfg.mux = kCLOCK_CAN2_ClockRoot_MuxSysPll3Out;
+	rootCfg.div = 6;
+	CLOCK_SetRootClock(kCLOCK_Root_Can2, &rootCfg);
+#endif
+
+#if DT_NODE_HAS_STATUS(DT_NODELABEL(flexcan3), okay)
+	/* Configure CAN3 using MuxSysPll3Out */
+	rootCfg.mux = kCLOCK_CAN2_ClockRoot_MuxSysPll3Out;
```

Comment:
Shouldn't it be `kCLOCK_CAN3_ClockRoot_MuxSysPll3Out`

Reply:
Sorry, I have corrected it. Thank you for reminding.